Gemfile

```diff
@@ -89,15 +89,12 @@ group :development do
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
-# Bulkrax
-group :bulkrax do
-  # Until b857a25becefc4bd960f25647033dfb2e899386e is in a major release, use this ref or a ref
-  # who's ancestors include this ref.
-  # rubocop:disable Metrics/LineLength
-  gem 'bulkrax', "~> 4.4", git: "https://github.com/samvera-labs/bulkrax.git", ref: "35970cee7b1e48d00c3fc838d2ca5790c09e2c9c"
-  # rubocop:enable Metrics/LineLength
-  gem 'willow_sword', git: 'https://github.com/notch8/willow_sword.git'
-end
+# Until b857a25becefc4bd960f25647033dfb2e899386e is in a major release, use this ref or a ref
+# who's ancestors include this ref.
+# rubocop:disable Metrics/LineLength
+gem 'bulkrax', "~> 4.4", git: "https://github.com/samvera-labs/bulkrax.git", ref: "35970cee7b1e48d00c3fc838d2ca5790c09e2c9c"
+# rubocop:enable Metrics/LineLength
+gem 'willow_sword', git: 'https://github.com/notch8/willow_sword.git'
```

Comment:
Should willow_sword also be removed since it isn't used?

Reply:
Do you know which version of bulkrax no longer requires it?

Reply:
@ShanaLMoore looks like this [commit](https://github.com/samvera-labs/bulkrax/commit/ab33927e5eb7b80d8737215c9e3f0dbf0707249d) removed it, but i'm unsure how to check what version it is

Reply:
Thanks for digging that up. It should be OK then. They are on version 4.4.0 which should include that commit/change. Thanks for pointing it out @laritakr

Reply:
Ps. I found that out by looking through Bulkrax's commit history